calibrators/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,119 @@
+| `apriltag_measurement_noise_translation`      | `double`              | `0.2`                            | The square of this value in meters is part of the input for the measurement noise covariance matrix (R) in the Kalman filter.                                                                                                                                           |
+| `apriltag_process_noise_translation`          | `double`              | `0.02`                           | The square of this value in meters is part of the input for the process noise covariance matrix (Q) in the Kalman filter.                                                                                                                                               |
+
+## Requirements
+
+### lidartag
+
+To perform camera-lidar calibration using this tool, it is necessary to prepare lidartags and lidars with intensity measures. To ensure that no objects obstruct the tag detection and to achieve the most stable detection possible, it is highly recommended to also prepare fixed mounts for these tags, as shown below.
+
+Note that the ones we used are lidartags of size 0.8 meters. This means the apriltags payload is 0.6 meters. We have also tried with lidartags of sizes 0.6 meters, but to use them the user needs to set several parameters by himself.
+
+<p align="center">
+    <img src="../docs/images/tag_based_pnp_calibrator/lidartag-mount.jpg"  alt="lidartag-mount" width="500">
+</p>
+
+## References
+
+[1] Jiunn-Kai (Bruce) Huang, Shoutian Wang, Maani Ghaffari, and Jessy W. Grizzle, "LiDARTag: A Real-Time Fiducial Tag System for Point Clouds," in IEEE Robotics and Automation Letters. Volume: 6, Issue: 3, July 2021.
+
+[2] G. Terzakis and M. Lourakis, "A Consistently Fast and Globally Optimal Solution to the Perspective-n-Point Problem" in ECCV 2020. Lecture Notes in Computer Science, vol 12346.
+
+## Known issues/limitations
+
+- The tool uses a basic OpenCV camera model for calibration (plumb bomb).
+
+## Pro tips/recommendations
+
+- During calibration, ensure that the lidar scan covers the tag, similar to the first example shown in the image below. However, if the tag resolution is low, as in the second example, and the lidar still detects the tag, it may still be acceptable but should be avoided when possible. The third example demonstrates a scenario where the lidar scan fails to cover the tag, resulting in the inability to detect the lidartag.
+
+<p align="center">
+    <img src="../docs/images/tag_based_pnp_calibrator/lidarscan_on_tag.jpg"  alt="lidarscan_on_tag" width="500">
+</p>
+
+- It is highly recommended to place the tag perpendicular to the calibration lidar as shown in the following image:
```

Comment:
Sorry, I missed this one before because I was seeing it in text format. The text says perpendicular to the lidar (which is correct), but the image is camera oriented

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/0505594af95ff8a48386cf516807d034faf5dfa8

Reply:
I thought the image was accurate. When we do the experiment before, didn't we always say the tag should face the camera? Also, this image is from the Sensor calibration説明会・研修 that you sent me, which means it is also not correct in that slide. 
Thanks

Reply:
No, it was always to face the lidar. Depending on the angle and on the surface, the beams do not bounce back strongly and the readings can become weird. 
If you are referring to the slides, please read both slide n34 and n35, I just confirmed that it is correctly explained

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/1a30887bbf64ddd0ee9fa33e7ab0eea9e98e1656 and https://github.com/tier4/CalibrationTools/pull/164/commits/1e3e98ec27d540586f015577fe2d93316f9ff8e9